src/configstore.ts

```diff
@@ -59,6 +66,16 @@ function getSettingsToAdd(config: any, label?: string, prefix?: string, tags?: T
     return settings;
 }
 
+function getSettingValue(value: any): string | undefined {
+    if (value === null || value === undefined) {
+        return undefined;

```

Comment:
>undefined [](start = 15, length = 9)

Why undefined and not null? If the key is present in the config doesn't that mean it's defined?

Reply:
The return value is passed to the SDK, which is defined as accepting 'strict | undefined'. That doesn't include null with the default strict type checking rules (fails a strict null check). Undefined felt like the closest value, but some other options considered:

1) Return null and bypass the strict null check when calling the SDK (by using the `!` operator). The end behavior is the same as passing undefined, but bypassing the type declaration didn't feel desirable.

2) Return a different non null value (like empty string). That felt different than the intent of the config.

Let me know if you'd prefer either of those options or a different one.

Reply:
Well it was definitely defined by the user so somehow transforming it into undefined is problematic. The way I see it is that we either transform it into an empty string or enforce the user to always provide a defined non-null value. Using our current portal import behavior as a reference I would go with converting to an empty string.

The less undefined we work with in code the better.

Reply:
Ok, I've changed this to convert to an empty string